templates/oiofs.cfg.j2

```diff
@@ -29,5 +30,8 @@
     "stats_server": "{{ mountpoint.stats_server }}",
 {% endif %}
     "sds_retry_delay": {{ mountpoint.sds_retry_delay | default(oiofs_mountpoint_default_sds_retry_delay) | int | to_json }},
-    "syslog_prefix": "{{ mountpoint.namespace | default(oiofs_mountpoint_default_namespace) }},oiofs,{{ mountpoint_id }}"
+    "syslog_prefix": "{{ mountpoint.namespace | default(oiofs_mountpoint_default_namespace) }},oiofs,{{ mountpoint_id }}",
+    "full_cache_timeout": {{ mountpoint.full_cache_timeout | default(oiofs_mountpoint_default_full_cache_timeout) | int | to_json }},
+    "active_mode": {{ mountpoint.active_mode | default(oiofs_mountpoint_default_active_mode) | bool | to_json }},
+    "chunk_readahead": {{ mountpoint.chunk_readahead | default(oiofs_mountpoint_default_chunk_readahead | int | to_json }}
```

Comment:
missing closing `)` for `default(...)`
rebase -i to squash into relevant commit, please